README.md

```diff
@@ -30,9 +30,9 @@
 
 ### 语言模型
 
-* `claude-v1.3` (默认使用): claude-v1 旧版本，效果稳定
-* `claude-instant-1`: claude 最新版 instant 模型
-* `claude-2`: claude 最新版模型
+* `claude 3 Haiku` Claude快速的模型
```

Comment:
Great update. It would be beneficial to simultaneously update the English documentation in 'docs/README_EN.md'.